src/components/main-page/HotPlaces.jsx

```
@@ -140,26 +120,27 @@ const HotPlaces = ({ accessToken, refreshAccessToken }) => {
-          selectedCategory === category.id
-            ? 'border-[#3288ff] text-blue-500 font-semibold'
-            : 'border-gray-300 text-gray-600'
-        }
-        hover:bg-gray-100 
-        text-[12px] px-[8px] py-[4px]
-        sm:text-base sm:px-2 sm:py-1
-      `}
+            ref={(el) => (buttonRefs.current[category.id] = el)}
+            onClick={() => {
+              setSelectedCategory(category.id);
+              buttonRefs.current[category.id]?.scrollIntoView({
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)